monitor/data_monitoring.py

```diff
@@ -64,8 +51,16 @@ def __init__(
         # slack client is optional
         self.slack_client = SlackClient.create_slack_client(self.slack_token, self.slack_webhook)
         self._download_dbt_package_if_needed(force_update_dbt_package)
+        self.elementary_database_and_schema = self.get_elementary_database_and_schema()
         self.success = True
 
+    def get_elementary_database_and_schema(self):
+        try:
+            database_and_schema = self.dbt_runner.run_operation('get_elementary_database_and_schema')[0]
```

Comment:
```suggestion
            database_and_schema = self.dbt_runner.run_operation(macro_name='get_elementary_database_and_schema')[0]
```
You can never know if run_operation will change, and the first argument will still be macro_name

Reply:
`run-operation` is a command that runs a macro.
I feel confident in using it as such and treating the first argument as the macro's name.